pkg/aws/sts.go

```diff
@@ -167,6 +168,75 @@ func UpgradeOperatorPolicies(reporter *rprtr.Object, awsClient Client, accountID
 	return nil
 }
 
+func BuildOperatorRoleCommandsFromCluster(
+	prefix string,
+	accountID string,
+	awsClient Client,
+	defaultPolicyVersion string,
+	credRequests map[string]*cmv1.STSOperator,
+	operatorRoles []*cmv1.OperatorIAMRole,
+) ([]string, error) {
+	commands := []string{}
+	for credrequest, operator := range credRequests {
+		_, operatorRoleARN := findOperatorRoleBySTSOperator(operatorRoles, operator)
+		if operatorRoleARN == "" {
+			continue
+		}
+		operatorRoleName, err := GetResourceIdFromARN(operatorRoleARN)
+		if err != nil {
+			return commands, err
+		}
+		policiesDetails, err := awsClient.GetAttachedPolicy(&operatorRoleName)
+		if err != nil {
+			return commands, err
+		}
+		if CountAttachedPoliciesInDetails(policiesDetails) > 1 {
```

Comment:
why are we checking this ?

Reply:
We actually already sort of do that already but we didn't error
We always have taken the 0, but if for some reason a non standard client is using more than 1 policy create by himself we might have gotten the wrong one

@vkareh mentioned we might want to error out for those who are way off the standard, so we wouldn't have to check anything else atm

Reply:
No i dont think we can do that because we are in progress of supporting installer break out.